root/static/scripts/edit/URLCleanup.js

```diff
@@ -5698,16 +5698,30 @@ const CLEANUPS: CleanupEntries = {
   },
   'tiktok': {
     match: [new RegExp('^(https?://)?(www\\.)?tiktok\\.com', 'i')],
-    restrict: [LINK_TYPES.socialnetwork],
+    restrict: [{...LINK_TYPES.streamingfree, ...LINK_TYPES.socialnetwork}],
     clean: function (url) {
       return url.replace(
-        /^(?:https?:\/\/)(?:www\.)?tiktok\.com\/@([\w.]+)(?:[\/?&#].*)?$/,
+        /^(?:https?:\/\/)(?:www\.)?tiktok\.com\/@([\w.]+(?:\/video\/\d+)?)(?:[\/?#].*)?$/,
```

Comment:
It is generally fine to keep `&` in the list of possible separators as some websites use it instead of `?` despite the fact that it doesn’t follow the [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).

No need to keep `&` here as TikTok seems to follow the standard. This regexp has been probably initially copied from a rule for another website that has this kind of issue.